js/src/Collection/TodoCollection.js

```diff
@@ -0,0 +1,18 @@
+var Backbone = require('backbone');
+
+var TodoModel = require('../Model/TodoModel.js');
+
+var TodoCollection = Backbone.Collection.extend({
+    initialize: function(models, options){
+        this.id = options.id
+    },
+    model: TodoModel,
```

Comment:
I typically see this moved all the way to the top, so that the reader knows immediately what models this collection should hold. Also, could you add a comment. For properties and variables in the defaults dictionary, the comment is very similar:
```
/**
 * Comment if necessary.
 * @type {data type}
 */
```
So for example:
```
/**
 * The collection's model.
 * @type {TodoModel}
 */
```